events.json

```diff
@@ -41,6 +41,28 @@
         ],
         "achievements": []
     },
+    {
+        "title": "Hackergarten Paris January 2019",
+        "date": "2019-01-29",
+        "venue": "Pivotal France",
+        "address": "33, rue La Fayette 75009 Paris",
+        "links": [
+            {
+                "title": "Meetup",
+                "url": "https://www.meetup.com/Paris-Hackergarten/events/tvdrwpyzcbmc/"
+            }
+        ],
+        "achievements": [
+            {
+                "title": "First try and feedback on project riff",
+                "url": "https://projectriff.io/"
+            },
+            {
+                "title": "New comment style contributed to fbiville/headache",
+                "url": "https://github.com/fbiville/headache/pull/38/files"
+            },
```

Comment:
This extra comma failed the build

Reply:
TooMuchGolangException, let me fix that :)